src/modules/LanItems.rb

```diff
@@ -2770,22 +2750,15 @@ def export_udevs(devices)
             Ops.set(ay, ["s390-devices", device, "protocol"], protocol)
           end
           Ops.set(ay, ["s390-devices", device, "layer2"], true) if layer2
-          port0 = Convert.convert(
-            SCR.Execute(
-              path(".target.bash_output"),
-              Builtins.sformat(
-                "port0=$(ls -l /sys/class/net/%1/device/cdev0);echo ${port0##*/}|tr -d '\n'",
-                device
-              )
-            ),
-            from: "any",
-            to:   "map <string, any>"
+          port0 = SCR.Execute(
+            path(".target.bash_output"),
+            Builtins.sformat(
+              "port0=$(/usr/bin/ls -l /sys/class/net/%1/device/cdev0); /usr/bin/echo ${port0##*/} | /usr/bin/tr -d '\n'",
```

Comment:
again the ```echo``` question

Reply:
and same reply. BTW that shell formatting is hell.

Reply:
I found this: https://askubuntu.com/questions/960822/why-is-there-a-bin-echo-and-why-would-i-want-to-use-it

btw shell formatting is hell but not that deep as used to be formatting in ycp ;-)